typescript/sdk/src/contracts/contracts.ts

```diff
@@ -81,7 +83,8 @@ export function filterAddressesToProtocol(
 export function attachContracts<F extends HyperlaneFactories>(
   addresses: HyperlaneAddresses<F>,
   factories: F,
-): HyperlaneContracts<F> {
+): any {
```

Comment:
Why lose the type here?

Reply:
thanks, yep the type can be kept here. A lot of this code is still in flux, just opened this PR to share the e2e pipeline result with Nam